packages/fxa-payments-server/server/lib/flow-performance.js

```diff
@@ -0,0 +1,125 @@
+const os = require('os');
+const log = require('./logging/log')();
+
+const MAX_DATA_LENGTH = 100;
+const VERSION = 1;
+const HOSTNAME = os.hostname();
+const PERFORMANCE_TIMINGS = [
```

Comment:
Functions and constants duplicated from content server's `lib/flow-event.js`, like `PERFORMANCE_TIMINGS`, `limitLength`, and `isValidTime`, should instead be extracted to fxa-shared. This will ensure we don't fix a bug in one place, but not the other.